crates/muvm/src/guest/x11.rs

```diff
@@ -89,6 +84,11 @@ where
         // See https://doc.rust-lang.org/std/env/fn.set_var.html#safety
         env::set_var("XAUTHORITY", dst_path);
     }
+    thread::spawn(|| {
+        if catch_unwind(|| start_x11bridge(1)).is_err() {
+            eprintln!("x11bridge thread crashed, x11 passthrough will no longer function");
```

Comment:
This is a bit annoying for debugging, since I often restarted x11bridge without restarting the whole VM, and also often wanted to strace it without stracing everything. Can we add a command line argument or something to just start x11bridge?

Reply:
I would slightly prefer if we kept the command line parsing as simple as possible. Would a env variable be ok in this case?

Reply:
Sure, it's just for debug anyway.